src/test/java/tictactoe_game/PutMethodTest.java

```diff
@@ -0,0 +1,263 @@
+package tictactoe_game;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import tictactoe_game.model.Coordinate;
+import tictactoe_game.model.Element;
+import tictactoe_game.model.Grid;
+
+public class PutMethodTest {
+    @Test
+    @DisplayName("Check 'X' element's position on coordinate '0,0'")
```

Comment:
The display names are not expressive. It is hard to understand, what the test is testing. When thinking of such descriptions, don't try to be short just for the sake of being short. It is better to give long but descriptive messages and then narrow them down, rather than artificially shortening them by losing expressive power.